packages/web-components/src/avatar/avatar.stories.ts

```diff
@@ -1,5 +1,5 @@
 import { html } from '@microsoft/fast-element';
-import { type NewMeta as Meta, renderComponent, type StoryArgs, type StoryObj } from '../helpers.stories.js';
+import { Meta, renderComponent, type StoryArgs, type StoryObj } from '../helpers.stories.js';
```

Comment:
`Meta` is still a type (it should be updated in all stories in this PR)
 
```suggestion
import { type Meta, renderComponent, type StoryArgs, type StoryObj } from '../helpers.stories.js';
```